api/client/client.go

```diff
@@ -216,8 +240,7 @@ func (c *Client) AddFile(
 	if err = stream.Send(&pb.AddFileRequest{
 		Payload: &pb.AddFileRequest_Header_{
 			Header: &pb.AddFileRequest_Header{
-				Name:      filepath.Base(file.Name()),
-				ProjectID: projID,
+				Path: filePath,
```

Comment:
This path (and others used to interact with files) is of the form: `<folder_name>/path/to/file/name.ext`.